app/workers/education_form/create_daily_fiscal_year_to_date_report.rb

```diff
@@ -14,7 +14,11 @@ class CreateDailyFiscalYearToDateReport
     }.freeze
 
     FORM_TYPE_HEADERS = EducationBenefitsClaim::FORM_TYPES.map do |form_type|
-      ["22-#{form_type}", '', '']
+      if form_type == '1995s'
```

Comment:
Seems like this would be clearer if you just materialize the constant -- literally type it in -- instead of dynamically generating it here *and* in the `app/workers/education_form/create_daily_year_to_date_report.rb`.

This seems like a UI deviation in the CSV file from all the other similar examples, which runs the risk of this `if` turning into something more complex with each additional change for similar reasons.

Since you're generating it in two different places, I'm wondering if you can make it a constant in a single place

```
FORM_TYPE_HEADERS=[
"22-1990", "", "", 
"22-1995", "", "", 
...
"22-1995 STEM", "", ""
]
```

And if you have to generate them, let's see about generating them in a single place.

Reply:
Did the spreadsheet/csv here just become three columns ("22-1995 STEM", "", "") wider?

Reply:
Hi @johnpaulashenfelter,

The spreadsheet/csv still has the same amount of columns, the only change was to what was being output to the report, in this case the header name which I was setting using the `if` statement check in those two places as you mentioned, since it differed from the form type slightly ("1995 STEM" vs "1995s").

I agree performing that check in two places was not ideal. I took your advice and created `FORM_HEADERS` inside of `EducationBenefitsClaim` as a way to decouple the form header from the form type, in case we ever do find ourselves in a situation where we need to output a different header than what was being generated dynamically.

I pushed these changes in another commit.